### PR TITLE
Remove section with deprecated code (#1817)

### PR DESCRIPTION
**Pull request by andrisnoko**

### CODE REVIEW

File: site/src/paradox/FAQ.md [hunk 1]
```diff
@@ -248,21 +248,6 @@ sc.textFile(...)
 
 In this example, the `map`'s transform name is "MakeUpper" and the `filter`'s is "BigWords". If we later decided that we want to count 6 letter words as "big" too, then we can change it to `_.length > 5`, and because the transform name is the same the job can be updated on the fly.
 
-#### How do I read Pubsub input in a local pipeline?
```
[the remark this answers is not in the document]
Comment (andrisnoko) [reply site/src/paradox/FAQ.md]: Not that I can think of. Will have to look into it. There is this issue [BEAM-2715](https://issues.apache.org/jira/browse/BEAM-2715)

[the remark this answers is not in the document]
Comment (nevillelyh) [reply site/src/paradox/FAQ.md]: There's an follow up 2 weeks ago. Should we respond?
https://issues.apache.org/jira/browse/BEAM-2715?focusedCommentId=16795596&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16795596

[the remark this answers is not in the document]
Comment (andrisnoko) [reply site/src/paradox/FAQ.md]: +1 I missed the last response.

[the remark this answers is not in the document]
Comment (regadas) [reply site/src/paradox/FAQ.md]: Given that this seems to no longer be possible... I think we should remove this from docs and create an issue to track the upstream

**Pull request merged**